src/app.ts

```diff
@@ -98,7 +100,7 @@ export class App extends Construct {
   /**
    * Synthesizes all manifests to the output directory
    */
-  public synth(): void {
+  public synth(): any {
```

Comment:
Not sure I love the fact that the return _type_ is dependent on `YamlOutputType`. Can we just create another method called `synthYaml(): string` instead?

Thoughts?

Reply:
Sure, makes sense. New commit coming up.